FreeRTOS/FreeRTOS
# Remove dependencies between network interface and demo MPS2

Remove dependencies between network interface and demo MPS2

Description
-----------
Remove dependencies between network interface and demo MPS2
Fix Makefile to achieve that
Remove the CMSIS directory from the demo

Test Steps
-----------
make build the project successfully 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
